.github/workflows/publish_ci.yml

```
@@ -38,32 +42,43 @@ jobs:
         dotnet pack --no-build --nologo --configuration Release /p:IsNuget=true -o ../../nuget
     - name: Publish FSharp.Data.GraphQL.Shared project to GitHub
       run: |
-        cd src/FSharp.Data.GraphQL.Shared
-        dotnet nuget push ../../nuget/*Shared*.nupkg -s "github" -k ${{secrets.GITHUB_TOKEN}}
+        dotnet nuget push nuget/*Shared*.nupkg -s "github" -k ${{secrets.GITHUB_TOKEN}}
 
     - name: Pack FSharp.Data.GraphQL.Client project
       run: |
         cd src/FSharp.Data.GraphQL.Client
         dotnet pack --no-build --nologo --configuration Release /p:IsNuget=true -o ../../nuget
     - name: Publish FSharp.Data.GraphQL.Client project to GitHub
       run: |
-        cd src/FSharp.Data.GraphQL.Client
-        dotnet nuget push ../../nuget/*Client*.nupkg -s "github" -k ${{secrets.GITHUB_TOKEN}}
+        dotnet nuget push nuget/*Client*.nupkg -s "github" -k ${{secrets.GITHUB_TOKEN}}
 
     - name: Pack FSharp.Data.GraphQL.Server project
       run: |
         cd src/FSharp.Data.GraphQL.Server
         dotnet pack --no-build --nologo --configuration Release /p:IsNuget=true -o ../../nuget
     - name: Publish FSharp.Data.GraphQL.Server project to GitHub
       run: |
-        cd src/FSharp.Data.GraphQL.Server
-        dotnet nuget push ../../nuget/*Server*.nupkg -s "github" -k ${{secrets.GITHUB_TOKEN}}
+        dotnet nuget push nuget/*Server*.nupkg -s "github" -k ${{secrets.GITHUB_TOKEN}}
 
     - name: Pack FSharp.Data.GraphQL.Server.Middleware project
       run: |
         cd src/FSharp.Data.GraphQL.Server.Middleware
         dotnet pack --nologo --configuration Release /p:IsNuget=true -o ../../nuget
     - name: Publish FSharp.Data.GraphQL.Server.Middleware project to GitHub
       run: |
-        cd src/FSharp.Data.GraphQL.Server.Middleware
-        dotnet nuget push ../../nuget/*Server.Middleware*.nupkg -s "github" -k ${{secrets.GITHUB_TOKEN}}
+        dotnet nuget push nuget/*Server.Middleware*.nupkg -s "github" -k ${{secrets.GITHUB_TOKEN}}
+
+    - name: Edit template project before packing
+      run: |
+        & "./Edit template project for packing.ps1"
+      shell: pwsh
```

```suggestion
    - name: Prepare the sample project to be packed as a project template
      run: |
        & "./Prepare template project for packing.ps1"
      shell: pwsh
```
Переименуй PowerShell